### PR TITLE
fix: allow AWS credentials to be provided for local development

### PR DESCRIPTION
**Pull request by chejimmy**
# Description

This PR allows the local development environment to work with provided AWS credentials. After this PR, the application is able to retrieve and utilize asset data on dashboards.

# How Has This Been Tested?

e2e tests passing and validated dashboards manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Legal

This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Comment by dpportet** [comment]
Is there somewhere we can see the deployed application to make sure the cloud credentials weren't affected?

File: README.md [hunk 1]
```diff
@@ -31,12 +32,13 @@ Start using the application today to quickly create, manage, and view your IoT d
...
    yarn install
    ```
+1. Update the variables `CLIENT_AWS_ACCESS_KEY_ID`, `CLIENT_AWS_SECRET_ACCESS_KEY`, and `CLIENT_AWS_SESSION_TOKEN` under file `apps/core/.env` with your development AWS credentials.
 1. Start development server:
    ```sh
    yarn dev
    ```
 1. Application is available at URL: `http://localhost:3000`
-1. Log in with local Cognito credentials found at `apps/core/.cognito/db`
+1. Log in with local Cognito credentials found at `apps/core/.cognito/db/us-west-2_h23TJjQR9.json`
```
Comment (corteggiano) [inline README.md:41]: will this file name be dependent on the region of the AWS credentials?

Comment (chejimmy) [reply README.md:41]: No, All static for local dev purpose only.

**Review by dpportet — CHANGES_REQUESTED** [review]
Would it be possible to also have these creds be used for the [migration client](https://github.com/awslabs/iot-application/blob/main/apps/core/src/migration/service/migration.service.ts#L31)? Without these the migration will fail locally.

**Comment by chejimmy** [comment]
> Would it be possible to also have these creds be used for the [migration client](https://github.com/awslabs/iot-application/blob/main/apps/core/src/migration/service/migration.service.ts#L31)? Without these the migration will fail locally.

good callout
let's use the same `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` under local dev
what do you think?

**Comment by dpportet** [comment]
> > Would it be possible to also have these creds be used for the [migration client](https://github.com/awslabs/iot-application/blob/main/apps/core/src/migration/service/migration.service.ts#L31)? Without these the migration will fail locally.
> 
> good callout let's use the same `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` for both Client and Core under local dev what do you think?

Sounds good to me, for local testing you'd want to use the same creds for both so that makes sense.

**Review by dpportet — APPROVED** [review]
Manually validated the local dev creds work for migration and the regular dashboard sitewise calls.

**Pull request merged**